apps/backoffice-v2/public/locales/en/toast.json

```diff
@@ -151,5 +151,14 @@
   },
   "edit_collection_flow_state_transition": {
     "error": "Error occurred while transitioning to edit collection flow."
+  },
+  "kyb_and_ownership_assessment_creation": {
+    "success": "KYB & Ownership assessment created successfully.",
```

Comment:
Not sure we want to mix internal terminology with product. Assessment might not be clear enough for a customer.